thangs_importer.py

```diff
@@ -12,6 +12,7 @@
 from .config import get_config
 from .thangs_events import ThangsEvents
 from .thangs_login import ThangsLogin
+from .version_import import VerisonImport
```

Comment:
I suggest just something like `model_importer` as the name of this module.  The fact that it deals with different versions of blender is a detail that the code using it doesn't really need to worry about.

So, probably something like
```
from .model_importer import import_model
```